examples/plot_histogram_testing.py

```diff
@@ -196,3 +196,56 @@
 #   When using tolerances, the ``tolerance`` argument is taken as a relative
 #   tolerance. For more information, see the documentation for the
 #   ``base.assert_bin_heights()`` method.
+
+################################################################################
+# Testing the Histogram Midpoints
+# -------------------------------
+# So far, we have tested the histogram values as well as the number of bins
+# the histogram has. It may also be useful to test that the data bins cover
+# the range of values that they were expected to. In order to do this, we can
+# test the midpoints of each bin to ensure that the data covered by each
+# bin is as expected. This is tested very similarly to the bins values.
+# Simply provide ``assert_bin_midpoints()`` with a list of the expected
+# midpoints, and it will assert if they are accurate or not. In order to obtain
+# the midpoints in a PlotTester object, you can use ``get_bin_midpoints()``,
+# much like ``get_bin_values()``.
+#
+# For this example, we will create a plot tester object from a histogram plot,
```

Comment:
@nkorinek this is great - will you please consistently use "you" rather than "we"? I prefer you for vignettes and lessons!!